### DISCUSSION TITLE
How do I keep index.log when render for debugging purpose

### DISCUSSION THREAD

**Discussion by martinlardi** [quarto-dev/quarto-cli#6664] [Q&A] [answered]
### Description

I want to see index.log
When render quarto book, it generate index.log that will be deleted after, if the render is successful. Is there a way to keep index.log even when render is successful ?

**Comment by mcanouil** [comment]
Mostly because I don't want the o assume, could you elaborate what is your project as "book" is only a type of project which can have various formats.

**Comment by martinlardi** [comment]
quarto book render to pdf. 
While rendering, it create index.log which got deleted after the pdf is generated. I want to keep the index.log

**Reply by mcanouil** [reply, marked as answer]
See <https://quarto.org/docs/reference/formats/pdf.html#latexmk> in particular `latex-clean`.

**Comment by martinlardi** [comment]
It works thanks !
`latex-clean: false`
